src/main/java/it/aboutbits/springboot/toolbox/boot/type/CustomTypeConfiguration.java

```diff
@@ -0,0 +1,78 @@
+package it.aboutbits.springboot.toolbox.boot.type;
+
+import it.aboutbits.springboot.toolbox.reflection.util.ClassScannerUtil;
+import it.aboutbits.springboot.toolbox.type.CustomType;
+import it.aboutbits.springboot.toolbox.type.jackson.CustomTypeDeserializer;
+import it.aboutbits.springboot.toolbox.type.jackson.CustomTypeSerializer;
+import it.aboutbits.springboot.toolbox.type.mvc.CustomTypePropertyEditor;
+import jakarta.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.autoconfigure.jackson.Jackson2ObjectMapperBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.InitBinder;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Configuration
+public class CustomTypeConfiguration {
+    public static final String LIBRARY_BASE_PACKAGE_NAME = "it.aboutbits.springboot.toolbox";
+
+    private String[] packageNamesToScan;
+    private ClassScannerUtil.ClassScanner classScanner;
+
+    public void setAdditionalTypePackages(String[] additionalTypePackages) {
+        var tmp = new ArrayList<String>();
+        tmp.add(LIBRARY_BASE_PACKAGE_NAME);
+        tmp.addAll(Arrays.asList(additionalTypePackages));
+
+        this.packageNamesToScan = tmp.toArray(new String[0]);
+
+        classScanner = ClassScannerUtil.getScannerForPackages(packageNamesToScan);
+    }
+
+    @PostConstruct
+    public void init() {
+        log.info("CustomTypeConfiguration enabled. Scanning: {}", Arrays.toString(packageNamesToScan));
+    }
```

Comment:
Does this log statement work?

Isn't the method `setAdditionalTypePackages` called after construction of the instance?